crates/holochain/src/core/workflow/publish_dht_ops_workflow.rs

```diff
@@ -42,10 +45,11 @@ pub async fn publish_dht_ops_workflow(
 ) -> WorkflowResult<WorkComplete> {
     let mut complete = WorkComplete::Complete;
     let to_publish = publish_dht_ops_workflow_inner(db.clone().into(), agent.clone()).await?;
+    let to_publish_count: usize = to_publish.values().map(Vec::len).sum();
 
     // Commit to the network
-    tracing::info!("publishing {} ops", to_publish.len());
-    let mut success = Vec::new();
+    info!("publishing {} ops", to_publish_count);
```

Comment:
Because this is a hashmap the count was for the number of keys rather than the sum across values. This explains why we see things like 'publishing 4 ops` then `published 7 ops`. This should clean up the logging